run/deployment-previews/check_status.py

```diff
@@ -239,10 +241,21 @@ def set(
         return
     # [END_EXCLUDE]
 
+    # Create unique sha for specific set status run
+    status_sha = hashlib.sha256(json.dumps({
+        "dry_run": dry_run,
+        "project_id": project_id,
+        "region": region,
+        "service": service,
+        "repo_name": repo_name,
+        "commit_sha": commit_sha,
+        "pull_request": pull_request,
+        "revision_url": revision_url,
+    }, sort_keys=True).encode("utf-8")).hexdigest()
     commit.create_status(
         state="success",
         target_url=revision_url,
-        context="Deployment Preview",
+        context=f"Deployment Preview: {status_sha[:7]}",
```

Comment:
```suggestion
        context=f"Deployment Preview for {service}",
```

This is the change I suggest in https://github.com/GoogleCloudPlatform/python-docs-samples/issues/8063#issuecomment-1154608372, which means you don't need to create the `status_sha` either.